gci/feeds.py

```diff
@@ -0,0 +1,34 @@
+import os.path
+from ruamel.yaml import YAML
+import markdown2
+
+from django.contrib.syndication.views import Feed
+from community.git import get_deploy_url
+
+
+class LatestTasksFeed(Feed):
+    title = 'GCI tasks feed'
+    link = get_deploy_url() + '/gci/rss/'
+    description = 'GCI tasks ordered by modification time.'
+
+    def items(self):
+        yaml = YAML()
+        with open(os.path.join('private', 'tasks.yaml')) as f:
```

Comment:
you need to load `_site/tasks.yaml`.  `private/tasks.yaml` is data which is private and can not be published.